recipes/nagl/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "nagl" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SimonBoothroyd/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: cca4b5c975405d720368a337f2908586ef44185934f7ac56c98bb4d078151153
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

Would be good to go ahead and add the entry point for the CLI:
```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
  entry_points:
    - nagl = nagl.cli:cli
```